atom/browser/api/atom_api_session.cc

```diff
@@ -305,6 +307,12 @@ void SetProxyInIO(scoped_refptr<net::URLRequestContextGetter> getter,
       BrowserThread::UI, FROM_HERE, callback);
 }
 
+void ClearHSTSDataInIO(scoped_refptr<net::URLRequestContextGetter> getter) {
+  auto request_context = getter->GetURLRequestContext();
+  request_context->transport_security_state()->
```

Comment:
sorry, I think I confused things with two separate comments. The IO thread part was just for future information. I think we actually want to call `storage_partition->GetNetworkContext()->ClearNetworkingHistorySince` which can be called on the UI thread and also calls `url_request_context_->http_server_properties()->Clear`. You can get the storage partition with
```
storage_partition =
        BrowserContext::GetDefaultStoragePartition(profile_);
```